docs/features/execution_payload_retention.md

```diff
@@ -12,19 +12,37 @@ This allows access via the API (or optionally direct DB/Elasticsearch querying)
 
 ## Payload record cleanup
 
-To reduce storage requirements, a CloudWatch rule (`{stack-name}-dailyExecutionPayloadCleanupRule`) triggering a daily run of the provided cleanExecutions lambda has been added.  This lambda will remove all 'completed' and 'non-completed' payload records in the database that are older than the specified configuration.
+<!-- To reduce storage requirements, a CloudWatch rule (`{stack-name}-dailyExecutionPayloadCleanupRule`) triggering a daily run of the provided cleanExecutions lambda has been added.  This lambda will remove all 'completed' and 'non-completed' payload records in the database that are older than the specified configuration. -->
```

Comment:
I assume that this wasn't intentional? Do we still want this content?

Reply:
Yes, and yes. 

511636c2e